### DISCUSSION TITLE
Ensamble model (lightgbm + PaddlePaddle)

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#60] [BaiduKDDCup2022] [answered]
Hi everyone, thanks for the submission manual . But how can we make a submission that uses: xgboost, lightgbm and PaddlePaddle models?. I think right now it can't be possible. Thanks in advanced.

**Comment by ramber1836** [comment, marked as answer]
It is possible.
Since XGBoost, LightGBM, as well as PaddlePaddle are all supported by the
system environment.

**Reply by fersebas** [reply]
So which framework I have to choose in submission (Base, PaddlePaddle,...) I can only select one....

**Comment by ramber1836** [comment]
In general, it depends on which framework plays a more important role in your model.
